worker/src/main.rs

```diff
@@ -248,13 +252,11 @@ fn worker<NF: NodeApiFactory, THA: TokioHandleAccessor>(
     println!("MRENCLAVE={}", mrenclave.to_base58());
     let eid = enclave.geteid();
 
-    let node_api = node_api_factory
-        .create_api()
-        .set_signer(AccountKeyring::Alice.pair());
+    let node_api = node_api_factory().set_signer(AccountKeyring::Alice.pair());
 
     WorkerAccessorImpl::reset_worker(Worker::new(
         config.clone(),
-        node_api,
+        node_api.clone(),
```

Comment:
These are only minor cosmetics I came across, they have actually nothing to do with my suggestion otherwise. :)